frontend/src/app/Notification.jsx

```diff
@@ -0,0 +1,53 @@
+import { useEffect, useState } from "react";
```

Comment:
More of a nit but I think all the notifications files can be in their own folder.

Reply:
Fixed in this commit: https://github.com/izuchukwumba/peered/pull/10/commits/a885f96c46e5113c233efece82a61a2a639ca89d